.github/workflows/bumpdevversion.yml

```
@@ -0,0 +1,115 @@
+on:
+  pull_request:
+    branches:
+      - devel
+  # Schedule runs on 9am every Tuesday
+  schedule:
+    - cron: '0 9 * * 2'
+
+jobs:
+  bump-dev-version:
+    name: Bump GMT development version
+    runs-on: ubuntu-latest
+    if: github.repository != 'conda-forge/gmt-feedstock'
```

One problem is anyone with a fork of this repo will get this action running (possibly causing repeat PRs). Wouldn't it be better to have it run **only** on `GenericMappingTools/gmt-feedstock`?

---

Yes, I'll change this when time comes to merge. Currently it's running on my personal fork at `weiji14/gmt-feedstock` so that things will run.
```suggestion
    if: github.repository == 'GenericMappingTools/gmt-feedstock'
```